src/ape_plugins/_cli.py

```diff
@@ -116,26 +117,49 @@ def add(cli_ctx, plugin, version, skip_confirmation):
         subprocess.call([sys.executable, "-m", "pip", "install", "--quiet", plugin])
 
 
+def _get_config_error() -> ConfigError:
+    expected_format = "plugins:\n  - name: <plugin-name>\n    version: <plugin-version>  # optional"
+    return ConfigError(f"Config item mis-configured. Expected format:\n\n{expected_format}\n")
+
+
+def _get_plugin_names(item: Dict) -> Tuple[str, str]:
```

Comment:
Naming here doesn't match what it returns. suggest `_extract_module_and_package_names`